angr/analyses/cfg/cfg_fast.py

```diff
@@ -3813,6 +3823,12 @@ def _generate_cfgnode(self, cfg_job, current_function_addr):
                     distance_ = section.vaddr + section.memsize - real_addr
                     distance = min(distance_, VEX_IRSB_MAX_SIZE)
                 # TODO: handle segment information as well
+                if segment is not None:
```

Comment:
Likewise, if the address falls within a section, do we still care about its segment?